protected/humhub/modules/tour/tests/codeception/acceptance/AutoTourCest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2018 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\tour\tests\codeception\acceptance;
+
+use tour\AcceptanceTester;
+
+class AutoTourCest
```

Comment:
Interesting would be cases that:

- The Tour is automatically triggered on the first login and not shown again on the second login.
- Is not triggered on the first login if disabled via Module::autoStartOnNewAccounts

Reply:
I've made the appropriate changes for the accetnas test, but unfortunately I'm currently running MacOS on an arm64 and can't run either selenium or phantomjs to test it, as they aren't yet optimized for this processor.